docs/content/config/advanced/auth-ldap.md

```diff
@@ -6,37 +6,188 @@ title: 'Advanced | LDAP Authentication'
 
 Getting started with ldap and this mailserver we need to take 3 parts in account:
 
-- `postfix`
-- `dovecot`
-- `saslauthd` (this can also be handled by dovecot)
+- `postfix` for incoming & outgoing email
+- `dovecot` for accessing mailboxes
+- `saslauthd` for SMTP authentication (this can also be delegated to dovecot)
 
 ## Variables to Control Provisioning by the Container
 
 Have a look at the [`ENVIRONMENT.md`][github-file-env] for information on the default values.
 
-!!! example "postfix"
+### `LDAP_QUERY_FILTER_*`
+Those variables contain the LDAP lookup filters for postfix, using `%s` as the placeholder for the domain or email address in question. This means that...
 
-    - `LDAP_QUERY_FILTER_USER`
-    - `LDAP_QUERY_FILTER_GROUP`
-    - `LDAP_QUERY_FILTER_ALIAS`
-    - `LDAP_QUERY_FILTER_DOMAIN`
-    - `LDAP_QUERY_FILTER_SENDERS`
+- ...for incoming email, the domain must return an entry for the `DOMAIN` filter (see [`virtual_alias_domains`](http://www.postfix.org/postconf.5.html#virtual_alias_domains)).
+- ...for incoming email, the inboxes which receive the email are chosen by the `USER`, `ALIAS` and `GROUP` filters.
+    - The `USER` filter specifies personal mailboxes, for which only one should exist per address, for example `(mail=%s)` (also see [`virtual_mailbox_maps`](http://www.postfix.org/postconf.5.html#virtual_mailbox_maps))
+    - The `ALIAS` filter specifies aliases for mailboxes, using [`virtual_alias_maps`](http://www.postfix.org/postconf.5.html#virtual_alias_maps), for example `(mailAlias=%s)`
+    - The `GROUP` filter specifies the personal mailboxes in a group (for emails that multiple people shall receive), using [`virtual_alias_maps`](http://www.postfix.org/postconf.5.html#virtual_alias_maps), for example `(mailGroupMember=%s)`
+    - Technically, there is no difference between `ALIAS` and `GROUP`, but ideally you should use `ALIAS` for personal aliases for a singular person (like `ceo@example.org`) and `GROUP` for multiple people (like `hr@example.org`).
+- ...for outgoing email, the sender address is put through the `SENDERS` filter, and only if the authenticated user is one of the returned entries, the email can be sent.
+    - This only applies if `SPOOF_PROTECTION=1`.
+    - If the `SENDERS` filter is missing, the `USER`, `ALIAS` and `GROUP` filters will be used in in a disjunction (OR).
+    - To for example allow users from the `admin` group to spoof any sender email address, and to force everyone else to only use their personal mailbox address for outgoing email, you can use something like this: `(|(memberOf=cn=admin,*)(mail=%s))`
 
-!!! example "saslauthd"
+???+ example "Example Code: A really simple `LDAP_QUERY_FILTER` configuration, using only the user filter and allowing only `admin@*` to spoof any sender addresses"
```

Comment:
```suggestion
???+ example

    A really simple `LDAP_QUERY_FILTER` configuration, using only the _user filter_ and allowing only `admin@*` to spoof any sender addresses.
```

Title seemed a bit too long imo. Suggest to use default "Example" title as it's the only one for this section, and moving the description into the admonition body.